backend/app/Http/Controllers/CategoryController.php

```diff
@@ -31,12 +31,9 @@ public function store(Request $request)
     }
 
     // Display Description
-    public function show(Request $request)
+    public function show($id)
     {
-        $request->validate([
-            'title' => ['required'],
-        ]);
-        $category = Category::where('title', $request->title)->first();
+        $category = Category::where('id', $id)->first();
```

Comment:
Maybe you can do this if you're getting only one data via ID from Category
`
 $category = Category::findOrFail($id);
`